CDCgov/prime-reportstream
# update upload-covid-19 schema for ordering facility fields

This PR updates the ordering_facility_city, ordering_facility_zip_code and ordering_facility_phone_number elements for the upload-covid-19 schema.

**If you are suggesting a fix for a currently exploitable issue, please disclose the issue to the prime-reportstream team directly outside of GitHub instead of filing a PR, so we may immediately patch the affected systems before a disclosure. See [SECURITY.md/Reporting a Vulnerability](https://github.com/CDCgov/prime-reportstream/blob/master/SECURITY.md#reporting-a-vulnerability) for more information.**

Test Steps:
1. create local org dendi with schema upload-covid-19 (or get settings for org dendi from staging)
2. Using attached test file run ./prime data --input-client dendi --input <test-file> --output-dir <output-file-location> --route
3. Ensure that output file has values in ordering_facility_city, ordering_facility_zip_code and ordering_facility_phone_number

## Changes
- Updates ifNotPresent mapper for ordering_facility_city, ordering_facility_zip_code and ordering_facility_phone_number to use testing lab equivalent fields.

### Testing
- [x ] Tested locally?
- [x ] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?

[Dendi-test.csv](https://github.com/CDCgov/prime-reportstream/files/10987597/Dendi-test.csv)
